cardscan-base/src/main/res/layout/bouncer_private_activity_scan_card.xml

```diff
@@ -76,12 +76,30 @@
         android:layout_marginRight="16dp"
         android:gravity="center"
         android:text="@string/card_scan_position_card"
-        android:textSize="17sp"
         android:textColor="@android:color/white"
+        android:textSize="17sp"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toBottomOf="@+id/cardRectangle" />
 
+    <TextView
+        android:id="@+id/enterCardManuallyButton"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="16dp"
+        android:layout_marginTop="8dp"
+        android:layout_marginEnd="16dp"
+        android:layout_marginBottom="8dp"
+        android:gravity="center"
+        android:onClick="onClick"
+        android:text="@string/card_scan_enter_card_manually"
+        android:visibility="visible"
```

Comment:
this is redundant.

Reply:
is there a reason this is defaulting to visible? should this be `GONE` unless set to visible by the code?